backend/index.js

```diff
@@ -35,6 +36,12 @@ app.set('view engine', 'njk')
 nunjucksSetup(app)
 phaseNameSetup(app, config)
 
+app.use(
+  helmet({
```

Comment:
this could go into a setup file?

Reply:
We already have helmet in `setupWebSecurity.js`

Reply:
https://github.com/ministryofjustice/prisonstaffhub/blob/main/backend/setupWebSecurity.js#L15